src/deephaven_riptable/__init__.py

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
+#
+"""Deephaven Riptable Integration Package (deephaven_riptable) supports the conversion between Deephaven tables and
+Riptable Dataset."""
+
+__version__ = "0.1.0"
```

Comment:
I think the whole point of `setuptools-scm` is so you don't include this file? (I'd also be in favor of removing `setuptools-scm`, because I do think it's a bit more magic.)

Reply:
agree